modules/calcite/src/main/java/org/apache/ignite/internal/processors/query/calcite/exec/exp/RexImpTable.java

```diff
@@ -1093,12 +1087,11 @@ else if (exprVal == SqlJsonEmptyOrError.ERROR) {
             newOperands.add(dfltValOnEmpty);
             newOperands.add(errorBehavior);
             newOperands.add(dfltValOnError);
-            Class clazz = method.getDeclaringClass();
-            expression = EnumUtils.call(null, clazz, method.getName(), newOperands);
-
-            final Type returnType =
-                translator.typeFactory.getJavaClass(call.getType());
-            return EnumUtils.convert(expression, returnType);
+            List<Expression> argValList0 =
```

Comment:
I think we should not skip Ignite's codestyle when taking side code. Line breaks between different operations, single lines, etc.

Reply:
It's a lot of changes in thirt-party code. It can lead to difficulties in synchronization, so I wouldn't apply Ignite codestyle to this code, only autoformating.